cfme/services/myservice.py

```diff
@@ -1,86 +1,158 @@
-    ])
-
-edit_tags_form = Form(
-    fields=[
-        ("select_tag", ui.Select("select#tag_cat")),
-        ("select_value", ui.Select("select#tag_add"))
-    ])
 
+class MyServicesView(BaseLoggedInPage):
+    def in_explorer(self):
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ['Services', 'MyServices'])
 
-class MyService(Updateable, Navigatable):
-    """Create,Edit and Delete Button Groups
+    @property
+    def is_displayed(self):
+        return self.in_explorer and self.configuration.is_displayed and not self.myservice.is_dimmed
 
-    Args:
-        service_name: The name of service to retire.
-        vm_name: Name of vm in the service.
-        retirement_date: Date to retire service.
-    """
+    @View.nested
+    class myservice(Accordion):  # noqa
+        ACCORDION_NAME = "Services"
 
-    def __init__(self, service_name, vm_name=None, appliance=None):
-        self.service_name = service_name
-        self.vm_name = vm_name
-        Navigatable.__init__(self, appliance=appliance)
+        tree = ManageIQTree()
 
-    def get_detail(self, properties=None, icon_href=False):
-        """ Gets details from the details infoblock
+    configuration = Dropdown('Configuration')
+    policy_btn = Dropdown('Policy')
+    lifecycle_btn = Dropdown('Lifecycle')
+    download_choice = Dropdown('Download')
 
-        Args:
-            *properties: An InfoBlock title, followed by the Key name e.g. "Relationships", "Images"
-            icon_href: Boolean indicating to return icon_href instead of text
-        Returns: A string representing the contents of the InfoBlock's value.
-        """
-        navigate_to(self, 'Details')
-        if icon_href:
-            return InfoBlock.icon_href(*properties)
-        else:
-            return InfoBlock.text(*properties)
+
+class ServiceRetirementForm(MyServicesView):
+    title = Text('#explorer_title_text')
+
+    retirement_date = Calendar('retirementDate')
+    retirement_warning = BootstrapSelect('retirement_warn')
+
+
+class ServiceEditForm(MyServicesView):
+    title = Text('#explorer_title_text')
+
+    name = Input(name='name')
+    description = Input(name='description')
+
+
+class SetOwnershipForm(MyServicesView):
+    title = Text('#explorer_title_text')
+
+    select_owner = BootstrapSelect('user_name')
+    select_group = BootstrapSelect('group_name')
+
+
+class EditTagsForm(MyServicesView):
+    title = Text('#explorer_title_text')
+
+    select_tag = BootstrapSelect('tag_cat')
+    select_value = BootstrapSelect('tag_add')
+
+
+class MyServiceDetailView(MyServicesView):
+    title = Text("#explorer_title_text")
+
+    properties_detail = SummaryTable(title='Properties')
+    lifecycle_detail = SummaryTable(title='Lifecycle')
+    relationships_detail = SummaryTable(title='Relationships')
+    vm_detail = SummaryTable(title='Totals for Service VMs ')
+    smart_mgmnt_detail = SummaryTable(title='Smart Management')
```

Comment:
We should probably use `mgmt` as it will be in line with our other references to "management".